app/src/main/res/values/styles.xml

```diff
@@ -180,18 +180,32 @@
         <item name="android:windowExitAnimation">@anim/fade_out</item>
     </style>
 
-    <style name="ThemeIndependentMaterialGreyButton" parent="Widget.MaterialComponents.Button.TextButton">
+    <!-- UI button styling -->
+    <style name="NeutralButton" parent="Widget.MaterialComponents.Button.TextButton">
+        <item name="iconTint">@color/button_text_color</item>
+        <item name="iconPadding">8dp</item>
+        <item name="iconGravity">textStart</item>
+        <item name="android:textAlignment">center</item>
+        <item name="android:layout_width">match_parent</item>
+        <item name="android:layout_height">48dp</item>
         <item name="android:textStyle">bold</item>
         <item name="android:textAllCaps">false</item>
         <item name="backgroundTint">@color/grey_button_color</item>
         <item name="android:textColor">@color/button_text_color</item>
         <item name="rippleColor">?secondaryText</item>
+        <item name="android:letterSpacing">0</item>
```

Comment:
Why did the letter spacing need to be specified here and/or is different from the other styles?

Reply:
The letters looked super spread out when I didn't change this. It's defined as "0" in several places thorughout the app, and I think it should be a consistent thing. I will confirm with UX and file a follow up for addressing this, but I think 0 is fine for now?